routes/cube_routes.js

```diff
@@ -73,6 +74,323 @@ const { ensureAuth, csrfProtection, flashValidationErrors, jsonValidationErrors
 
 router.use(csrfProtection);
 
+const fetchLands = {
```

Comment:
Can we put the bot rating code into a helper module like draftutil? These functions seem like they would fit well with the draft creation code there.